client-js/app/scripts/registry.ts

```diff
@@ -9,182 +11,10 @@ export interface RegistryListing {
   vals: Array<string>;
 }
 
-export interface RegistryApi {
```

Comment:
Oh I think I get it now, becuase hte actual sendRequest is specifc to the websocket jsonRPC